homeassistant/components/zwave_js/light.py

```diff
@@ -186,7 +194,6 @@ def color_mode(self) -> str | None:
 
     @property
     def is_on(self) -> bool:
-        """Return true if device is on (brightness above 0)."""
         return self._state
```

Comment:
this should be reinstated to the default again
in the brightness attribute there can be a fallback to the last known brightness but only if the device is transitioning to on

Reply:
yeah, we technically could probably get rid of all the optimistic state stuff...? I tested a combined solution, but I can try testing out with just the **Last known brightness** changes and see what happens.

Reply:
Ok, the one reason you'd want to keep this here: after restarting HA, **Last known brightness** is `None`, so it will fallback to the current functionality the first time you turn the lights on from HA (the whole toggle on, off and then on again).

This is something I could live with, but again, _I_ would much prefer to persist the value through a restart and use that, _even if it's wrong_. (Which it'd be a lot more likely to be wrong if HA was down.)

Could always make this behavior a toggle-able setting!